test/check-storage-reclaim

```diff
@@ -236,7 +236,7 @@ class TestStorageReclaimSpace(VirtInstallMachineCase):
                 ("", "extended"),
                 ("2.5GB", "logical"),
             ],
-            is_efi=True
+            is_mbr=True
```

Comment:
Shouldn't this be `is_mbr=False` ?

Reply:
extended partitions are only valid as part of MBR, this code path in partition_disk, was setting up msdos on disk instead of GPT and then supported creating extended partitions on this.

Btw I did not change the logic, I just changed the variable name, as it was missleading.

Reply:
Okay, I was probably confused by the test name containing EFI compared to the newly added `_testExtendedPartition_partition_disk`